src/npc_class.cpp

```diff
@@ -267,7 +267,7 @@ distribution distribution::dice_roll( int sides, int size )
     } );
 }
 
-distribution distribution::operator+( const distribution &other ) const
+distribution distribution::sum( const distribution &other ) const
```

Comment:
There could definitely be a better name than sum (good point about sum vs add), and I'll admit I might have gotten carried away with distribution, that one is pretty reasonable on second look.

And yes, the lack of idiomatic usage is the majority of the problem,. if it doesn't do precisely what you expect based on the operator it's a detriment to readability rather than a benefit, and in general giving it a named method is safer because you aren't inheriting strong assumptions about how it is supposed to work.